Frontend/app/problem/display/displayProblem.html

```diff
@@ -24,9 +24,15 @@
         </ul>
       </span>
     </div>
-    <p class="lead" *ngIf="problem">
-      Equation I'm trying to solve: <em>{{problem}}</em>
-    </p>
+    <div *ngIf="problem">
+      <p class="lead">
+        <img id="mentii-avatar" src="../../../favicon/mentii-avatar2.png" alt="Mentii Logo">
```

Comment:
@jonathanmcdaniel moved it to /images in root
@aaronmsegal I tried, this was the closest thing I got to our color from the site i used